packages/i18n/src/locales/en.i18n.json

```diff
@@ -4398,6 +4398,12 @@
   "SAML_Custom_Private_Key": "Private Key Contents",
   "SAML_Custom_Provider": "Custom Provider",
   "SAML_Custom_Public_Cert": "Public Cert Contents",
+  "SAML_Custom_Signature_Algorithm": "Signature Algorithm",
+  "SAML_Custom_Signature_Algorithm_description": "What Algorithm to use when signing requests and responses from Rocket.Chat",
+  "SAML_Custom_Signature_Algorithm_SHA1": "SHA1",
+  "SAML_Custom_Signature_Algorithm_SHA256": "SHA256",
+  "SAML_Custom_Signature_Algorithm_SHA384": "SHA384",
+  "SAML_Custom_Signature_Algorithm_SHA512": "SHA512",
```

Comment:
these are not translatable, they won't change depending on the language, so no need to have i18n keys for them